projects/igniteui-angular/src/lib/splitter/splitter-bar/splitter-bar.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="igx-splitter-bar"
+<div tabindex="0" class="igx-splitter-bar"
```

Comment:
The main igx-splitter-bar element should be focusable, not the first child. TabIndex can be set via a HostBinding for the root element.